packaging/makeself/install-or-update.sh

```diff
@@ -214,7 +214,7 @@ run chown -R ${NETDATA_USER}:${NETDATA_GROUP} /opt/netdata
 
 progress "fix plugin permissions"
```

Comment:
I'd call this _changing ownership and setting setuid_

Reply:
ok, lets change it